### ISSUE TITLE
have a list for excluded folders, like all archive

### ISSUE THREAD

**Issue by opto** [opto/nostalgy-xpi#103]

**Comment by opto** [comment]
you can do that by using the regex - that can have substrings that should not occur

**Comment by mcepl** [comment]
> you can do that by using the regex - that can have substrings that should not occur

Could I get some example or tutorial for idiots how to do it?

**Comment by s-renier-taonix-fr** [comment]
We've got an example in "Usage directions for Nostalgy++" :
"Even if not using regex, Nostalgy++ now allows multiple search criteria. For example, 'in no' will find inbox/software/nostalgy. 'ar no' will find archive/software/nostalgy. Just using 'no' will find both. Two or more criteria or parts of foldernames are allowed. Don't type the '! "

But it's not good enough for me because I keep the structure of my folder in my archive.
And my folders are not in the inbox.
mail@example.com
|_ Inbox
|_ Archives
  | 2021
    |_ Customers
      |_ Customer A
      |_ ...
      |_ Customer Z
    |_ Perso
    |_ ...
  | 2020
    |_ Customers
      |_ Customer A
      |_ ...
      |_ Customer Z
    |_ Perso
    |_ ...
...
|_ Customers
  |_ Customer A
  |_ ...
  |_ Customer Z
|_ Perso
|_ ...

**Comment by tonycanike** [comment]
I agree with @s-renier-taonix-fr .    I use the "out of the box" Archive functionality built into Thunderbird, and it puts them in a folder tree as @s-renier-taonix-fr describes.     The Archive folder tree is essentially inactive and I don't use Nostalgy++ functionality to access it.   

I believe I'd like a new feature in the Nostalgy++ Preferences that would allow someone to exclude the Archive folder tree (or perhaps a regex to apply to folder names to exclude)  from the folder matching used by the S, C, G, etc. commands, and their Shift variants.

**Comment by tonycanike** [comment]
And I believe this enhancement request is a duplicate of #69.

**Comment by tonycanike** [comment]
I believe this could be implemented by having a new configuration setting in the "Completion" tab of the Nostalgy Preferences in which a allow-regex string could be entered.  [Similar to an allow-list, or whitelist, but it's a regex.) If this configuration setting was enabled, folder completion would need to match both the allow-regex string _and_ the regex string the user entered interactively after using a S, G, B, etc. command.

Alternatively, or even in addition to the allow-regex, there could be a setting to enter a deny-regex.   If the folder matched the deny-regex, it would not be displayed even if it matched the regex string the user entered interactively.

Thank you.
